Dockerfile

```diff
@@ -21,8 +21,6 @@ COPY proto/ ./proto
 RUN npm run build
 
 COPY src/favicon.ico ./dist/
-COPY src/mmo/assets/ ./dist/mmo/assets
```

Comment:
this has been moved to Webpack via CopyWebpackPlugin

Reply:
cool